util/reggen/countermeasure.py

```diff
@@ -0,0 +1,71 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Sequence
+
+from .lib import check_keys, check_str, check_list
+
+# TO BE DEFINED
+CM_ASSETS = [
+    'FOO'
+]
+
+# TO BE DEFINED
+CM_TYPES = [
+    'BAR'
+]
+
+
+class CounterMeasure:
+    def __init__(self, name: str, desc: str):
+        self.name = name
```

Comment:
Would it make more sense to store `name_split` here instead of the glommed-together version? (And maybe have a `get_name()` function that glued things back together)

Reply:
Sure, let me split that out.